CLAUDE.md

```
@@ -207,6 +388,14 @@ test/.../domain/{domain-name}/
     └── {Entity}ServiceIntegrationTest.java  # 통합 테스트
 ```
 
+**예시: VocabularyBook 도메인**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)